### PR TITLE
added info about DICOM

### PR DESCRIPTION
**Pull request by CPernet**
as per the steering group discussion 19 January 2023

### CODE REVIEW

**Comment by CPernet** [comment]
@robertoostenveld @arokem @ericearl is that reflecting the main point accurately?

File: _pages/bep_guide.md [hunk 1]
```diff
@@ -149,6 +149,10 @@ extension proposals should embrace these rather than trying to come up with
 alternative standards. The BIDS extension proposal should therefore inventorize
 and review past and existing work that may be relevant to the BEP.
 
+### Not considering DICOM fields
```
Comment (ericearl) [inline _pages/bep_guide.md:152]:
```suggestion
### Not considering fields defined by other standards
```

Meant to reflect the state of BIDS not being just about DICOM.

File: _pages/bep_guide.md [hunk 2]
```diff
@@ -149,6 +149,10 @@ extension proposals should embrace these rather than trying to come up with
...
+
+Many of the modalities we use have an associated DICOM standard. While BIDS is not specifically about data format, many metadata information are stored in data files and there is often no good reason for using a different name than the DICOM one. It is thus reasonable to [check what DICOM has developped](https://www.dicomstandard.org/) and see if an overlap can be made. In a similar fashion, when relevant, we recommend having a `source` folder in the example datasets with DICOM files (again empty data but with a header -- make sure to remove personal data tags if taken from real data)
```
Comment (ericearl) [inline _pages/bep_guide.md:154]:
```suggestion
Many of the modalities we use have an associated standard, like DICOM for instance. While BIDS is not specifically about data format, many metadata information are stored in data files and there is rarely a good reason for using a different name than one from other established standards. In using DICOM it is reasonable to [check what DICOM has already developed](https://www.dicomstandard.org/) and see if there is overlap. In a similar fashion, when relevant, we recommend having a `source/` folder in example datasets including DICOM files (empty data but with a header, removing any personal data tags if taken from "real" data).
```

Adding some clarity and a small amount of changes.

**Review by ericearl — APPROVED** [review]
If you can please consider those two suggestions I made, I am fine to approve this PR.

**Comment by neurolabusc** [comment]
Nice and concise. Thanks!

File: _pages/bep_guide.md [hunk 3]
```diff
@@ -149,6 +149,10 @@ extension proposals should embrace these rather than trying to come up with
 alternative standards. The BIDS extension proposal should therefore inventorize
 and review past and existing work that may be relevant to the BEP.
 
+### Not considering DICOM fields
+
+Many of the modalities we use have an associated standard, like DICOM for instance. While BIDS is not specifically about data format, many metadata information are stored in data files and there is rarely a good reason for using a different name than one from other established standards. In using DICOM it is reasonable to [check what DICOM has already developed](https://www.dicomstandard.org/) and see if there is overlap. In a similar fashion, when relevant, we recommend having a `source/` folder in example datasets including DICOM files (empty data but with a header, removing any personal data tags if taken from "real" data).
```
Comment (Remi-Gau) [inline _pages/bep_guide.md:154]: do we mean to add a `source` folder or is this the `sourcedata` folder of BIDS?

https://bids-specification.readthedocs.io/en/latest/common-principles.html#source-vs-raw-vs-derived-data

**Review by CPernet — COMMENTED** [review]
making changes

**Review by CPernet — COMMENTED** [review]
sourcedata changed

**Review by Remi-Gau — APPROVED** [review]

**Review by sappelhoff — APPROVED** [review]

**Comment by sappelhoff** [comment]
Thanks!

**Pull request merged**